source/common/json/json_validator.h

```diff
@@ -0,0 +1,17 @@
+#pragma once
+
+#include "envoy/json/json_object.h"
+
+namespace Json {
+
+class JsonValidator {
+public:
+  /*
```

Comment:
nit: move this comment to class level, you are also missing a '*' in the comment opening.